### DISCUSSION TITLE
Issue when trying to import a CSV

### DISCUSSION THREAD

**Discussion by bcarney66** [Sequel-Ace/Sequel-Ace#1959] [General]
I'm having a small issue when trying to import a CSV file to a new table.  Sequel-Ace adds a carriage return after the last field name and before the ` which causes the table creation to fail.  Running Version 4.0.13 Build 20062:

```
An error occurred while trying to add the new table 'NewTable' by

CREATE TABLE `NewTable` (
`Field1` VARCHAR(255), 
`Field2` VARCHAR(255), 
`Field3
` VARCHAR(255)).

MySQL said: Incorrect column name 'Field3
'

```
<img width="294" alt="Screen Shot 2024-01-24 at 10 40 36 AM" src="https://github.com/Sequel-Ace/Sequel-Ace/assets/10159409/60c3f18d-cf77-4bb7-a91c-a26e24b81433">

**Comment by bcarney66** [comment]
It appears to only happen when importing CSVs from the Mac version of Excel, so I'm assuming it's an Excel issue.

**Comment by samuelbradshaw** [comment]
I wonder if this is related to [New line gets imported with last CSV column when importing from CSV](https://github.com/Sequel-Ace/Sequel-Ace/issues/231)?
